docs/api_v0.yml

```diff
@@ -652,7 +652,11 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Comment"
-
+    ConfigUpsert:
```

Comment:
maybe just `SiteConfig` ?

`upsert` is the operation, we're describing the payload